standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnStore.java

```diff
@@ -104,6 +104,23 @@ enum MUTEX_KEY {
   String DID_NOT_INITIATE_RESPONSE = "did not initiate";
   String REFUSED_RESPONSE = "refused";
 
+  static final char INITIATED_STATE = 'i';
```

Comment:
those constants are already present in TxnHandler, did you forget to remove them from the original place

Reply:
my bad. done now